pkgs/development/libraries/catch2/3.nix

```diff
@@ -7,13 +7,13 @@
 
 stdenv.mkDerivation rec {
   pname = "catch2";
-  version = "3.3.2";
+  version = "3.4.0";
 
   src = fetchFromGitHub {
     owner = "catchorg";
     repo = "Catch2";
     rev = "v${version}";
-    hash = "sha256-t/4iCrzPeDZNNlgibVqx5rhe+d3lXwm1GmBMDDId0VQ=";
+    hash = "sha256-EikouBS3VxcxbHGvab0rQzh2Q7oHam7BbniCv7LfrLs=";
```

Comment:
I don't have the same hash:
```
error: hash mismatch in fixed-output derivation '/nix/store/kzna5978hkkm0ll7zrazkmw97cmpf08z-source.drv':
         specified: sha256-EikouBS3VxcxbHGvab0rQzh2Q7oHam7BbniCv7LfrLs=
            got:    sha256-DqGGfNjKPW9HFJrX9arFHyNYjB61uoL6NabZatTWrr0=

Reply:
Thanks for checking!

Weird. I created the hash with the following commands:

`nix-prefetch-url --type sha256 https://github.com/catchorg/Catch2/archive/refs/tags/v3.4.0.tar.gz`

which returns

`1fxcvyrbz0kqdv0nwsh7p91pcf235fynkbvidhqifmxp2jw2ha8j`

which i converted with

`nix hash to-sri --type sha256 1fxcvyrbz0kqdv0nwsh7p91pcf235fynkbvidhqifmxp2jw2ha8j`

to

`sha256-EikouBS3VxcxbHGvab0rQzh2Q7oHam7BbniCv7LfrLs=`

but i fixed the commit with your hash.

Reply:
I have no idea how we should compute them. Usually, I'll just increment a random number inside the old, and start the build. 
build tools will then fail and give me the right one :)

Reply:
Thank you! :)

Reply:
You have to use `nix-prefetch-url --unpack`.